Latex/hwSolutions.tex

```diff
@@ -60,5 +60,16 @@
 \showanswers
 
 \include{./chapter01/problemSet01}
+\include{./chapter02/problemSet02}
+\include{./chapter03/problemSet03}
+\include{./chapter04/problemSet04}
+\include{./chapter05/problemSet05}
+\include{./chapter06/problemSet06}
+\include{./chapter07/problemSet07}
+\include{./chapter08/problemSet08}
+\include{./chapter09/problemSet09}
+\include{./chapter10/problemSet10}
```

Comment:
like this (if you ever see this comment)

Reply:
# Realization
You can continue working on a branch even after its been merged.  Hmmm...
And everything is markdown :bowtie: